### PR TITLE
integration/docker: add test to check yamux IO keep alive

### LINKED ISSUE #310: Add test  for yamux: keepalive failed: i/o deadline reached
Add test for the issue https://github.com/kata-containers/agent/issues/231

### PR DESCRIPTION
**Pull request by devimc**
Check that yamux connection write timeout never occurs.
Connection write timeout occurs only if yamux default
configuration is used to create the client.

Depends-on: github.com/kata-containers/proxy#71

fixes #310

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: integration/docker/pause_test.go [hunk 1]
```diff
@@ -37,3 +40,39 @@ var _ = Describe("pause", func() {
...
+	})
+
+	Describe("pause, wait and unpause a container", func() {
+		Context("check yamux IO connection", func() {
+			It("should keep alive", func() {
+				_, _, exitCode = dockerRun("-td", "--name", id, Image, "sh")
+				Expect(0).To(Equal(exitCode))
+				_, _, exitCode = dockerPause(id)
+				Expect(0).To(Equal(exitCode))
+				time.Sleep(45 * time.Second)
```
Comment (jcvenegas) [inline integration/docker/pause_test.go]: could you explain why 45 sec is needed  and added it a comment if needed?

**Comment by devimc** [comment]
waiting for https://github.com/kata-containers/proxy/pull/71

**Comment by chavafg** [comment]
lgtm

File: integration/docker/pause_test.go [hunk 2]
```diff
@@ -37,3 +40,43 @@ var _ = Describe("pause", func() {
 		})
 	})
 })
+
+// To get more info about this test, see https://github.com/kata-containers/agent/issues/231
```
Comment (jodh-intel) [inline integration/docker/pause_test.go]: Good use of comments!

**Pull request merged**